rosenpass/src/protocol/protocol.rs

```diff
@@ -122,64 +192,281 @@ pub type PeerNo = usize;
 /// Not in scope of this struct:
 ///
 /// - handling of external IO (like sockets etc.)
+///
+///
+///
+/// # Example
+///
+/// See the example on how to use this CryptoServer without [Self::poll] in [crate::protocol].
+///
+/// See [Self::poll] on how to use a CryptoServer with poll.
 #[derive(Debug)]
 pub struct CryptoServer {
+    /// The source of most timing information for the Rosenpass protocol
+    ///
+    /// We store most timing information in the form of f64 values, relative to a point stored in
+    /// this field.
     pub timebase: Timebase,
 
-    // Server Crypto
+    /// Static Secret Key Mine (our secret key)
     pub sskm: SSk,
+    /// Static Publi Key Mine (our public key)
```

Comment:
Typo: Static Public Key

Reply:
Can you push a fix to my branch directly?

Reply:
Done.